content/en/News/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: News
+description: Stay updated with our latest news and announcements.
+menu:
+  main: { weight: 50 }
+outputs: [HTML, RSS]
+type: docs
```

Comment:
We'll want this to be a type other than `docs` eventually, but this will do for now.

Or, if you want, you can look at this as a possible model / example:

- https://github.com/open-telemetry/opentelemetry.io/blob/main/content/en/status.md

Reply:
I have worked a lot in writing the shortcode for news.html and this is looking consistent throughout the website with type docs according to me, but if you still think this is looking odd i will change.

Reply:
Yes, I'd like the page to look more like the News page of the current site. For example, from a page-design perspective, there is no need for a left-nav given that the new News page doesn't have any sections. There is also little need for the "Edit the page" link given that the news is contain in a data file.